proto/v1/sql_review_service.proto

```diff
@@ -0,0 +1,115 @@
+import "google/protobuf/field_mask.proto";
+import "google/protobuf/timestamp.proto";
+import "v1/org_policy_service.proto";
+
+option go_package = "generated-go/v1";
+
+service SQLReviewService {
+  rpc CreateSQLReview(CreateSQLReviewRequest) returns (SQLReview) {
+    option (google.api.http) = {
+      post: "/v1/sqlReviews"
+      body: "sql_review"
+    };
+    option (google.api.method_signature) = "";
+  }
+
+  rpc ListSQLReviews(ListSQLReviewsRequest) returns (ListSQLReviewsResponse) {
+    option (google.api.http) = {get: "/v1/sqlReviews"};
+    option (google.api.method_signature) = "";
+  }
+
+  rpc GetSQLReview(GetSQLReviewRequest) returns (SQLReview) {
+    option (google.api.http) = {get: "/v1/{name=sqlReviews/*}"};
+    option (google.api.method_signature) = "name";
+  }
+
+  rpc UpdateSQLReview(UpdateSQLReviewRequest) returns (SQLReview) {
+    option (google.api.http) = {
+      patch: "/v1/{sql_review.name=sqlReviews/*}"
+      body: "sql_review"
+    };
+    option (google.api.method_signature) = "sql_review,update_mask";
+  }
+
+  rpc DeleteSQLReview(DeleteSQLReviewRequest) returns (google.protobuf.Empty) {
+    option (google.api.http) = {delete: "/v1/{name=sqlReviews/*}"};
+    option (google.api.method_signature) = "name";
+  }
+}
+
+message ListSQLReviewsRequest {
+  // The maximum number of sql review to return. The service may return fewer than
+  // this value.
+  // If unspecified, at most 50 sql review will be returned.
+  // The maximum value is 1000; values above 1000 will be coerced to 1000.
+  int32 page_size = 1;
+
+  // A page token, provide this to retrieve the subsequent page.
+  //
+  // When paginating, all other parameters provided to `ListSQLReviews` must match
+  // the call that provided the page token.
+  string page_token = 2;
+}
+
+message ListSQLReviewsResponse {
+  // The sql review from the specified request.
+  repeated SQLReview sql_reviews = 1;
+
+  // A token, which can be sent as `page_token` to retrieve the next page.
+  // If this field is omitted, there are no subsequent pages.
+  string next_page_token = 2;
+}
+
+message CreateSQLReviewRequest {
+  // The sql review to create.
+  SQLReview sql_review = 1 [(google.api.field_behavior) = REQUIRED];
+}
+
+message UpdateSQLReviewRequest {
+  // The sql review toupdate.
+  //
+  // The name field is used to identify the sql review to update.
+  SQLReview sql_review = 1 [(google.api.field_behavior) = REQUIRED];
+
+  // The list of fields to update.
+  google.protobuf.FieldMask update_mask = 2;
+}
+
+message GetSQLReviewRequest {
+  // The name of the sql review to retrieve.
+  // Format: sqlReviews/{uid}
+  string name = 1 [(google.api.field_behavior) = REQUIRED];
+}
+
+message DeleteSQLReviewRequest {
+  // The name of the sql review to delete.
+  // Format: sqlReviews/{uid}
+  string name = 1 [(google.api.field_behavior) = REQUIRED];
+}
+
+message SQLReview {
+  // The name of the sql review to retrieve.
+  // Format: sqlReviews/{uid}
```

Comment:
what about sqlReviewConfig or sqlReviewRule or something like that?
Just SQLReview sounds like we are doing SQL review.

Reply:
LGTM

Reply:
How about ReviewConfig? We end up with doing more than SQL stuff.

Reply:
Working on `ReviewConfig`